R/server.R

```diff
@@ -186,3 +209,25 @@ check_not_interactive <- function(call = caller_env()) {
     )
   }
 }
+
+has_connected_sessions <- function() {
+  tryCatch(
+    {
+      aio <- nanonext::send_aio(
+        the$server_socket,
+        "",
+        mode = "raw",
+        timeout = 10L
+      )
+      nanonext::call_aio(aio)
+      !nanonext::is_error_value(aio$result)
+    },
+    error = function(e) FALSE
+  )
+}
```

Comment:
The reversion to running things in the server _after_ a session disappears doesn't actually work! Reverting to draft.